lib/engine/game/g_18_india/step/track.rb

```diff
@@ -1,15 +1,109 @@
 # frozen_string_literal: true
 
 require_relative '../../../step/track'
+require_relative 'gauge_change_border'
 require_relative 'railhead_tracker'
 
 module Engine
   module Game
     module G18India
       module Step
         class Track < Engine::Step::Track
+          include GaugeChangeBorder
           include RailheadTracker
 
+          # modified to add remove gauge change option in phase IV
+          def actions(entity)
+            return [] unless entity == current_entity
+            return [] unless @round.removed_gauge.empty?
+            return [] if entity.company? || !can_lay_tile?(entity)
+
+            actions = %w[lay_tile pass]
+            actions << 'remove_border' if may_remove_gauge_change?(entity)
+            actions
+          end
+
+          def setup
+            super
+            @round.removed_gauge = []
+          end
+
+          def round_state
+            super.merge({ removed_gauge: [] })
+          end
+
+          def description
+            tile_lay = get_tile_lay(current_entity)
+            return 'Lay Track' unless tile_lay
+
+            if tile_lay[:lay] && tile_lay[:upgrade]
+              @game.phase.name == 'IV' ? 'Lay/Upgrade Track OR Remove Gauge Change Marker' : 'Lay/Upgrade Track'
+            elsif tile_lay[:lay]
+              'Lay Track'
+            else
+              'Upgrade Track'
+            end
+          end
+
+          # ------ Show a note that gauge changes may be removed ------
+
+          def note_text_block
+            [
+              'May remove a Gauge Change Marker as a track action.',
+              'Click on a connected marker to remove it.',
+            ]
+          end
```

Comment:
See comments on assets/app/view/game/note.rb

Also, it is probably worth adding help/note text onto the running routes step, to indicate you don't need to click on the gauge change markers, but they still do count as stops.

Reply:
Converted this to use `help` view, it was straightforward to use help.  
Thanks for pointing out that class. 

I added help text to route step as suggested.